pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -976,3 +992,33 @@ kubelet --help | grep -- "--cgroup-driver"
 
 	return rc == 0 && err == nil
 }
+func (h *Helper) configureHostDockerTrust(registryURL string) error {
+	glog.V(1).Info("Copy master CA to host docker certificates location")
+	certPath := GetRegistryCertPath(registryURL)
+	if _, err := os.Stat(filepath.Dir(certPath)); os.IsNotExist(err) {
+		err := os.Mkdir(filepath.Dir(certPath), hostDockerCertPermissions)
```

Comment:
creating the directory should happen in the container. You can't assume that you have any access to the host except through Docker.

Reply:
so long cmd passed to the container. make sense.